examples/single/bundle/src/App.js

```diff
@@ -43,14 +56,18 @@ class App extends Component {
   }
 
   async componentDidMount() {
-    const session = await getSession();
     if (window.location.pathname === "/popup") {
       this.state.status = "popup";
       setTimeout(() => window.close(), 2000);
-    } else if (!session) {
+    } else if (!this.state.session.isLoggedIn) {
```

Comment:
Tiny point, but when an 'if' statement has an 'else' I'd always test for positive logic, i.e. remove the '!' (not) from the 'if' statement and reverse the then/else blocks.

Reply:
Fixed in c0d4fbddecd4014475237b33501ffb6163a99422